src/plugin/home/Resources/styles/home.less

```diff
@@ -14,6 +14,54 @@
 
   margin: 0 floor((@grid-gutter-width / 2) - 5px);
 
+  .top-tabs {
+    display: flex;
+    padding-left: 0;
+    margin-bottom: 0;
+    list-style: none;
+  }
+
+  .top-item.dropdown {
+    position: relative;
+
+    .top-tab {
+      padding: @padding-large-vertical 25px @padding-large-vertical @padding-large-horizontal;
+
+      &::before {
+        display: inline-block;
+        margin-left: .255em;
+        vertical-align: .255em;
+        content: "";
+        border-top: .3em solid;
+        border-right: .3em solid transparent;
+        border-bottom: 0;
+        border-left: .3em solid transparent;
+        right: 10px;
+        top: 45%;
+        position: absolute;
+      }
+    }
+
+    &:hover {
+      .dropdown-tabs {
+        display: block;
```

Comment:
This will cause accessibility issues on touchscreens.

Reply:
Thanks, we will take a closer look at this at a later stage. The submenu is currently hidden by default.

Reply:
We prevented the potential accessibility issues by showing an additional button - for expanding/collapsing the dropdown menu. Now the submenu is triggered by an "onClick" event rather than on hover.